lib/charms/layer/aws.py

```diff
@@ -855,3 +875,17 @@ def _cleanup_policy(policy_arn):
         log('Deleted IAM policy {}', policy_arn)
     except DoesNotExistAWSError:
         pass
+
+
+def _cleanup_subnet_tags(region, subnet_id, tags):
+    """
+    Cleanup an IAM instance-profile.
+    """
+    try:
+        _aws(*(['ec2', 'delete-tags'] +
+               ['--region', region] +
```

Comment:
AFAICT, delete-tags doesn't take a `region` param:

https://docs.aws.amazon.com/cli/latest/reference/ec2/delete-tags.html#synopsis

Reply:
For some reason, it's not documented, but it does take, and in fact requires, the region:

```
$ aws ec2 delete-tags --resources subnet-5096f67c subnet-cda9f385
You must specify a region. You can also configure your region by running "aws configure".
```